### DISCUSSION TITLE
Asking about an upgrade

### DISCUSSION THREAD

**Discussion by 328000** [vercel/community#4474] [Feature Question] [answered]
### Question

Hello
I have been using Vercel for over 2 years now. And i really like it. Its sooooo easy. But my projects need [monitoring](https://vercel.com/dashboard/monitoring) . Is there any way i can get it for free.

**Comment by amyegan** [comment, marked as answer]
Hi @ZlatislavZ! Glad to hear you like Vercel. 😄 

If you have an open source project that you want to deploy with Vercel, you can [apply for sponsorship](https://vercel.com/guides/can-vercel-sponsor-my-open-source-project). Otherwise, the current price is $10 per billing cycle for your first 250,000 data points and an additional $9 per 1 million data points after that.

https://vercel.com/docs/observability/monitoring/limits-and-pricing

**Reply by 328000** [reply]
@amyegan I don't want exactly sponsoring cause vercel will want me to have high volume traffic and lots of views. I just want to host my hobby projects and test things on vercel cause I like it.

Thanks
